src/main/java/net/minestom/server/instance/block/BlockFace.java

```diff
@@ -77,4 +95,12 @@ public static BlockFace fromDirection(Direction direction) {
             case EAST -> EAST;
         };
     }
+
+    /**
+     * Returns the static accessor which caches all entries from the values call.
+     * @return the given array
+     */
+    public static @NotNull BlockFace[] getValues() {
+        return VALUES;
+    }
```

Comment:
Is that the same style as other enum micro optimizations from us ?

Reply:
Yes, it follows the same style as our other optimizations in enums